pypose/function/geometry.py

```diff
@@ -111,6 +111,38 @@ def point2pixel(points, intrinsics, extrinsics=None):
     return homo2cart(points @ intrinsics.mT)
 
 
+def pixel2point(pixels, depth, intrinsics):
+    r'''
+    Convert batch of pixels with depth into points (in camera coordinate)
+
+    Args:
+        pixels: (``torch.Tensor``) The 2d coordinates of pixels in the camera pixel
+            coordinate.
+            Shape has to be (..., N, 2)
+
+        depth: (``torch.Tensor``) The depths of pixels with respect to the optical
+            center of camera.
+            Shape has to be (..., N)
+
+        intrinsics: (``torch.Tensor``): The intrinsic parameters of cameras.
+            The shape has to be (..., 3, 3).
+
+    Returns:
+        ``torch.Tensor`` The associated 3D-points with shape (..., N, 3)
```

Comment:
Need an example in doc to show how to use this function. 
It is to show grammar example, not a real example. refer to [point2pixel](https://pypose.org/docs/main/generated/pypose.point2pixel/?highlight=point2pixel#pypose.point2pixel)